test/models/script_test.rb

```diff
@@ -278,6 +302,42 @@ class ScriptTest < ActiveSupport::TestCase
     FileUtils.rm path
   end
 
+  test 'should not import a script with a different version' do
+    begin
+      ENV['TEST_VERSION'] = MonitorApp::Application::VERSION.next
+      uuid    = SecureRandom.uuid
+      scripts = []
```

Comment:
```suggestion
      ENV['TEST_VERSION'] = MonitorApp::Application::VERSION.next

      uuid    = SecureRandom.uuid
      scripts = []
```

Solo para que queden alineados los `=` en grupos.

Reply:
me olvideeee mala mia jajaja